tests/admintools.test.js

```diff
@@ -0,0 +1,217 @@
+'use strict';
+
+require('should');
+var _ = require('lodash');
+var benv = require('benv');
+var read = require('fs').readFileSync;
+var serverSettings = require('./fixtures/default-server-settings');
+
+var nowData = {
```

Comment:
![Codacy](https://www.codacy.com/versioned/images/favicon.png) Issue found: ['nowData' is defined but never used.](https://www.codacy.com/app/Nightscout/cgm-remote-monitor/file/1032753275/issues/source?bid=2452379&fileBranchId=2878156#l9)